test/test_utils.py

```diff
@@ -591,5 +592,42 @@ def test_import_hipify(self):
         from torch.utils.hipify import hipify_python # noqa
 
 
+class TestBenchmarkUtils(TestCase):
+    def test_timer(self):
+        timer = benchmark_utils.Timer(
+            stmt="torch.ones(())",
+        )
+        median = timer.blocked_autorange(min_run_time=0.1).median
+        self.assertIsInstance(median, float)
+
+    def test_compare(self):
+        compare = benchmark_utils.Compare([
+            benchmark_utils.Timer(
+                "torch.ones((n,))", globals={"n": n},
+                description="ones", label=str(n)).timeit(3)
+            for n in range(3)
+        ])
+        compare.print()
+
+    def test_fuzzer(self):
+        fuzzer = benchmark_utils.Fuzzer(
+            parameters=[
+                benchmark_utils.FuzzedParameter(
+                    "n", minval=1, maxval=16, distribution="loguniform")],
+            tensors=[benchmark_utils.FuzzedTensor("x", size=("n",))],
+            seed=0,
+        )
+
+        expected_results = [
+            (0.7821, 0.0536, 0.9888, 0.1949, 0.5242, 0.1987, 0.5094),
+            (0.7166, 0.5961, 0.8303, 0.005),
+        ]
+
+        for i, (tensors, _, _) in enumerate(fuzzer.take(2)):
+            x = tensors["x"]
+            self.assertTrue(
```

Comment:
can you please use self.assertEqual here instead of torch.allclose in the statement?

Reply:
Done. I didn't realize assertEqual would take a non-scalar value. Thanks!

Reply:
Not only non-scalar, it would recursively go over lists or tuples you send it.